CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
  * `testdb` no longer accepts the `--test_mysql_uri` flag, and instead honours the
    `TEST_MYSQL_URI` ENV var. This makes it easier to blanket configure tests to use a
    specific test DB instance.
+ * Set GCS bucket to send build logs to in cloudbuild.yaml
```

Comment:
This isn't necessary to be in the CHANGELOG, it will covered by "migrated to Cloud Build" item.

Reply:
Removed